src/api/spec/components/request_decision_component_spec.rb

```diff
@@ -0,0 +1,128 @@
+RSpec.describe RequestDecisionComponent, :vcr, type: :component do
```

Comment:
Is there a reason this is using VCR?

Reply:
Nope.
@danidoni as you're the original author of this initial part, any insight? I'll drop cassettes otherwise

Reply:
I'm sorry, I don't remember. If there is no evident reason I guess we can remove it.